doc/offline-signing-tutorial.md

```diff
@@ -0,0 +1,259 @@
+
+In this example, we are going to generate a key on the offline wallet, and then send it 0.0009 BTC with an online(sending) wallet, then using the online wallet draft a transaction to send 0.0008 BTC to another online(receiving) wallet, then sign that with the offline wallet, and finally announce the signed transaction into the network using the online wallet.
+
+> [!NOTE]
+> Tested using Signet with a connected host machine and an offline docker container (representing the offline wallet) both running Signet, should also work with Regtest and Mainnet
+
+
+## Step 1: Sending money to the offline wallet
+
+### Create and Load Funds into the Online Sending Wallet
+
+1. On the online machine, create an online wallet named `online_sending_wallet` and load funds into it.
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -named createwallet wallet_name="online_sending_wallet"
+{
+  "name": "online_sending_wallet"
+}
+```
+
+2. Generate an address from the `online_sending_wallet` to receive funds.
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -rpcwallet="online_sending_wallet" getnewaddress
+tb1q5hg8nvt8frlcth57n3s2h3lfqp8he9s0ea23q4
+```
+
+3. Visit a faucet like https://signet.bc-2.jp to load funds into the generated address.
+
+
+### Create and Prepare the Offline Wallet
+
+1. On the offline machine, create an offline wallet named `offline_wallet`.
+
+```sh
+[offline]$ ./src/bitcoin-cli -signet -named createwallet wallet_name="offline_wallet"
+{
+  "name": "offline_wallet"
+}
+```
+
+2. Export the offline wallet descriptors to a JSON file named `descriptors.json`. This file will be exported to the online machine to create a watch-only wallet.
+```sh
+[offline]$ ./src/bitcoin-cli -signet -rpcwallet="offline_wallet" listdescriptors | jq -r '.descriptors' >> ~/descriptors.json
+```
+
+### Create the online "watch_only_wallet"
+
+1. On the online machine, create an online watch-only wallet named `watch_only_wallet` without private keys.
+- The `watch_only_wallet` wallet will be used to track transactions received to the `offline_wallet` and for creating PSBT transactions.
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -named createwallet wallet_name="watch_only_wallet" disable_private_keys=true blank=true
+{
+  "name": "watch_only_wallet"
+}
+```
+
+2. Import the `offline_wallet` descriptors to the online `watch_only_wallet` using the `descriptors.json` file created on the offline wallet
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -rpcwallet="watch_only_wallet" importdescriptors "$(cat ~/descriptors.json)"
+[
+  {
+    "success": true
+  },
+  {
+    "success": true
+  },
+  {
+    "success": true
+  },
+  {
+    "success": true
+  },
+  {
+    "success": true
+  },
+  {
+    "success": true
+  },
+  {
+    "success": true
+  },
+  {
+    "success": true
+  }
+]
+```
+
+### Send Funds to the Offline Wallet
+
+1. Generate a new address in the `offline_wallet` to receive funds.
```

Comment:
You should be able to generate receive address from the online wallet now! I feel like one of the best benefits of this setup is that once you create the private keys in the offline wallet, you shouldn't even need to turn that computer on until you are ready to sign something.